### ISSUE TITLE
Tests fail with 5.25.11 due to '.' not in @INC change

### ISSUE THREAD

**Issue by genehack** [tokuhirom/cpan-outdated#36]
```
% ./Build test
t/00_compile.t ............ ok
t/01_compare_version.t .... do "script/cpan-outdated" failed, '.' is no longer in @INC at t/01_compare_version.t line 7.
Died at t/01_compare_version.t line 7.
# Looks like your test exited with 2 before it could output anything.
t/01_compare_version.t .... Dubious, test returned 2 (wstat 512, 0x200)
Failed 7/7 subtests
t/02_permissive_filter.t .. do "script/cpan-outdated" failed, '.' is no longer in @INC at t/02_permissive_filter.t line 7.
Died at t/02_permissive_filter.t line 7.
# Looks like your test exited with 2 before it could output anything.
t/02_permissive_filter.t .. Dubious, test returned 2 (wstat 512, 0x200)
Failed 6/6 subtests

Test Summary Report
-------------------
t/01_compare_version.t  (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 7 tests but ran 0.
t/02_permissive_filter.t (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 6 tests but ran 0.
Files=3, Tests=1,  0 wallclock secs ( 0.03 usr  0.01 sys +  0.10 cusr  0.00 csys =  0.14 CPU)
Result: FAIL
```

**Comment by dolmen** [comment]
Fix in #37 is merged.

**Comment by dolmen** [comment]
Released as 0.32

Cc: @syohex
